backend/app/views/spree/admin/shared/linkables/_spree_taxon.html.erb

```diff
@@ -6,19 +6,17 @@
  <% end %>
  <%= f.field_container :linked_resource_id, class: ['form-group'] do %>
    <%= f.label :linked_resource_id, Spree.t('taxon') %>
-   <%= f.select :linked_resource_id, options_from_collection_for_select(resorce_holder || [] , id_holder, name_holder, active_id_holder || nil), { include_blank: true }, class: 'autocomplete' %>
+   <%= f.select :linked_resource_id,
+        options_from_collection_for_select(resorce_holder || [],
+                                           id_holder, name_holder, active_id_holder || nil),
+        { include_blank: true },
+        class: 'autocomplete',
+        data: { ac_name: 'taxons',
+                ac_placeholder: Spree.t('admin.navigation.seach_for_a_taxon'),
+                ac_clear: true,
+                ac_url: 'taxons_api_v2',
+                ac_return_attr: 'pretty_name' } %>
```

Comment:
```suggestion
        data: { 'autocomplete-name-value': 'taxons',
                'autocomplete-placeholder-value': Spree.t('admin.navigation.seach_for_a_taxon'),
                'autocomplete-clear-value': true,
                'autocomplete-url-value': 'taxons_api_v2',
                'autocomplete-return-attribute-value': 'pretty_name' } %>
```

Reply:
that's the Stimulus conventions :) 

https://stimulus.hotwire.dev/reference/values

this way we would be able to switch to stimulus quote quickly

Reply:
Hi @damianlegawiec,

The HTML output is kebab-case: `data-ac-name="products" `

```html
<select class="select2autocomplete select2-hidden-accessible" data-ac-name="products" data-ac-placeholder="Search for a Product" data-ac-clear="true" data-ac-url="products_api_v2" name="menu_item[linked_resource_id]" id="menu_item_linked_resource_id" data-select2-id="menu_item_linked_resource_id" tabindex="-1" aria-hidden="true"><option value="" label=" " data-select2-id="2"></option>
<option selected="selected" value="17" data-select2-id="3">V Neck T Shirt</option></select>
```

I'll update this now to use the full name `autocomplete` and end the in `-value` and then I think it should be fully compatible.

Reply:
```html
<select class="select2autocomplete select2-hidden-accessible" data-autocomplete-name-value="taxons" data-autocomplete-placeholder-value="Search for a Taxon" data-autocomplete-clear-value="true" data-autocomplete-url-value="taxons_api_v2" data-autocomplete-return-attr-value="pretty_name" name="menu_item[linked_resource_id]" id="menu_item_linked_resource_id" data-select2-id="menu_item_linked_resource_id" tabindex="-1" aria-hidden="true"><option value="" label=" " data-select2-id="2"></option>
<option selected="selected" value="21" data-select2-id="3">Categories -&gt; Streetstyle</option></select>
```